README.md

```diff
@@ -45,17 +45,20 @@ The document number will be read from the files by default.
 # - output (echo) the commands that would be applied without noop
 # Good for testing
 docsplit.sh --noop "$DIR/data/Simple.pdf" "$TEST_DIR/file"
+
 # Shows the found pages from the default regex.
 # Good when the OCR failed for some pages.
 # In that case you can use the output, insert or change the missed pages and use them instead.
 docsplit.sh --pages "$DIR/data/Simple.pdf" "$TEST_DIR/file"
+
 # Uses the given pages instead of searching for some
```

Comment:
So if `pages=` is provided then this input will be used instead of the regex?

What does the syntax `1:385` mean?

Reply:
@spier : The syntax is `page_number:extracted_content`. In this case the number 385 is found on page 1.

Reply:
Got it. Might be worth adding that to the docs as well.